sensu/plugins/check-inspec.rb

```diff
@@ -60,18 +60,21 @@ def run
     results = inspec(config[:controls], config[:attrs])
     passed = 0
     failed = 0
+    skipped = 0
     msg = ""
     results['controls'].each do |control|
       #puts control
       if control['status'] == "passed"
         passed += 1
+      elsif control['status'] == "skipped"
```

Comment:
I'll add a print message that utilizes `skipped`, `failed`, and `passed`. The `passed` variable isn't used either.